packages/storybook/stories/InputDate.mdx

```diff
@@ -0,0 +1,38 @@
+import { Canvas, Meta } from '@storybook/blocks';
+import * as DateInputStories from './InputDate.stories';
+
+<Meta of={DateInputStories} />
+# DateInput Component
+
+The `DateInput` component is used for form inputs with HTML5 date validation
+
+## Properties
+- **name** (`string`): Required. The input elements name, this will become the key of the input on form submission. Can be nested using dot notation ie (fieldset.email)
+- **value** (`string`): Nullable. String value to make the input element as a controlled component, can be used to set defaultValue. 
+- **defaultValue** (`string`): Nullable. String value to set the value of the input element without turning it into a controlled component.
+- **id** (`string`): Nullable. used to tag the input element with an Id. must be unique
+- **rows** (`number`): Nullable. determines how many lines does the text area have on render
+- **label** (`string`): Nullable. label text that appears on top of the input element
+- **placeholder** (`string`): Nullable. property to determine the placeholder text of the input element
+- **required** (`booleean`): Nullable. used to determine if field is required.
+- **disabled** (`boolean`): Nullable. used to determine if field is disabled.
+- **hidden** (`boolean`): Nullable. used to determine if field is hidden. Can be used to have prefilled content without being visible to the end user
+- **touched** (`boolean`): Nullable. used to determine if field has been touched.
+- **dirty** (`boolean`): Nullable. used to determine if field dirty
+- **onFocus** (`function`): Nullable. used to hook into html onFocus input event.
+- **onChange** (`function`): Nullable. used to hook into html onChange input event.
+
+### Variants:
+#### Default
+<Canvas of={DateInputStories.Default} />
+#### Disabled
+<Canvas of={DateInputStories.Disabled} />
+#### Required
+<Canvas of={DateInputStories.Required} />
+#### Hidden
+<Canvas of={DateInputStories.Hidden} />
+#### Touched
+<Canvas of={DateInputStories.Touched} />
+#### Dirty
+<Canvas of={DateInputStories.Dirty} />
+---
```

Comment:
@ianmarquez  How can we test the `dirty` state? Hmm should we use the boolean in `FormInput` in combination with the `error` style?

Reply:
so far we cannot test as of yet, the dirty state and touched state is not doing anything for now. Its just there to determine if field is touched or dirty. This state comes from the react-hook-forms framework. Im still in the midst of implementing it although its just something thats nice to have.